eq-author-api/utils/datastoreFileSystem.js

```diff
@@ -0,0 +1,55 @@
+const { omit, reject } = require("lodash");
+const fs = require("fs");
+const stringify = require("json-stable-stringify");
+
+const createQuestionnaire = async questionnaire => {
+  fs.writeFileSync(
```

Comment:
Can we use `async` versions so the differences aren't too big. https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api